docs/install/_osx.rst

```diff
@@ -31,9 +31,11 @@ To install Kolibri on Linux distributions other than Debian, as well as on MacOS
 	  chmod +x kolibri-installer-filename.pex
 	  ./kolibri-installer-filename.pex start
 
-	.. warning:: Make sure to **substitute the** ``kolibri-installer-filename.pex`` **with the exact name of the file you downloaded** in both commands. For example, if the name of the downloaded file is ``kolibri-v0.10.0.pex``, type that instead of ``kolibri-installer-filename.pex``. 
+	.. note:: Make sure to **substitute the** ``kolibri-installer-filename.pex`` **with the exact name of the file you downloaded** in both commands. For example, if the name of the downloaded file is ``kolibri-v0.11.0.pex``, type that instead of ``kolibri-installer-filename.pex``. 
 
 #. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
+   
+.. warning:: Remember that **PEX** package allows you to run Kolibri on your system **only while the process is active in the Terminal window**. If that process is stopped (by either pressing :guilabel:`Ctrl+C`, closing the Terminal, or restarting your system), you will need to repeat the above steps to start Kolibri again. User accounts, classes and groups you create, as well as the content channels you download, will be available every time you restart Kolibri. 
```

Comment:
This is to test how would it be to try using the `:guilabel:` for at least when referring to button elements in Kolibri docs. With proposed styling it renders like this:

![ledev18aut start running - oracle vm virtualbox_117](https://user-images.githubusercontent.com/1457929/47646830-6c558280-db75-11e8-8b25-49104e19ed22.png)

I tested the default RTD styling on just one page of Studio docs, but wasn't happy with the result:

![working with channels kolibri studio 0 1 0 documentation - mozilla firefox_207](https://user-images.githubusercontent.com/1457929/47647123-601df500-db76-11e8-95ca-960f207224f0.png)


In theory Sphinx documentation recommends using this directive when referring to all and any GUI element, but I find that it would overload some sections, so for now would favor using it just for buttons (not tabs, headings, checkboxes, etc.):

![ledev18aut start running - oracle vm virtualbox_118](https://user-images.githubusercontent.com/1457929/47646968-f1409c00-db75-11e8-9692-0c17f90df161.png)

Reply:
One point of reference: in github (and HTML) you can use `<kbd>...</kbd>` to indicate something that is typed. I've usually seen it done like this, with the '+' left in normal text: <kbd>Ctrl</kbd>+<kbd>c</kbd>

My recommendation for a convention in our docs would be:

* Use `guilabel` with all keyboard references: 
    > ... by either pressing <kbd>Ctrl</kbd>+<kbd>c</kbd>, closing the Terminal ...
* Use `guilabel` and capital letters for all buttons:
    > Click <kbd>SAVE CHANGES</kbd> to apply and finish.
* For everything else, use bold (or maybe italics?) and name the UI element:
    > Click the channel you want to modify in the **My Channels** tab.

Reply:
@indirectlylit @radinamatic Sphinx has a `:kbd:` too :) http://www.sphinx-doc.org/en/1.5.2/markup/inline.html#role-kbd